### PR TITLE
Added camera info

### PR DESCRIPTION
**Pull request by trym-b**
Added tests for:
    camera revision
    camera firmware_version
    camera model name
    camera serial number

### CODE REVIEW

File: test/test_camera.py [hunk 1]
```diff
@@ -44,3 +44,31 @@ def test_illegal_set_settings(file_camera):
 
     with pytest.raises(AttributeError):
         file_camera.settings = zivid.Settings()
+def test_revision(file_camera):
+    revision = file_camera.revision
+    assert revision is not None
+    assert isinstance(revision, str)
+    assert revision == "0.0"
```
[the remark this answers is not in the document]
Comment (trym-b) [reply test/test_camera.py]: as long as the test takes in a file_camera, this makes sense. When it is no longer a file_camera, we can change it.
It is a separate issue.

File: zivid/camera.py [hunk 2]
```diff
@@ -11,6 +11,12 @@ def __init__(self, internal_camera):
                 f"Unsupported type for argument internal camera: {internal_camera}, type: {type(internal_camera)}."
             )
         self.__camera = internal_camera
+        self.firmware_version = internal_camera.firmware_version
+        self.model_name = internal_camera.model_name
+        self.revision = (
+            f"{internal_camera.revision.major}.{internal_camera.revision.minor}"
+        )
+        self.serial_number = str(internal_camera.serial_number)
```
[the remark this answers is not in the document]
Comment (trym-b) [reply zivid/camera.py]: Need the to string of it, is not the right type. It is of _zivid.SerialNumber, not string

File: zivid/camera.py [hunk 3]
```diff
@@ -11,6 +11,12 @@ def __init__(self, internal_camera):
                 f"Unsupported type for argument internal camera: {internal_camera}, type: {type(internal_camera)}."
             )
         self.__camera = internal_camera
+        self.firmware_version = internal_camera.firmware_version
```
[the remark this answers is not in the document]
Comment (trym-b) [reply zivid/camera.py]: fixed: change it into a property

File: zivid/camera.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (nedrebo) [reply zivid/camera.py]: This feels wrong. If we do not want a type for SerialNumber, then we can just make it a string in the pybind11 layer. It is a class in C++ for us to keep the door open for member functions and for it to be strongly typed. If we do not need that in C++ then I propose to delete SerialNumber.* and update camera wrapper like this:

.def_property_readonly("serial_number", [](ReleasableCamera &camera){ return camera.serialNumber().toString(); )

File: zivid/camera.py [hunk 4]
```diff
@@ -5,12 +5,32 @@
...
                 f"Unsupported type for argument internal camera: {internal_camera}, type: {type(internal_camera)}."
             )
         self.__camera = internal_camera
+        self.model_name = internal_camera.model_name
+        self.revision = Camera.Revision(
+            internal_camera.revision.major, internal_camera.revision.minor
+        )
+        self.serial_number = str(internal_camera.serial_number)
+
+    @property
```
[the remark this answers is not in the document]
Comment (trym-b) [reply zivid/camera.py]: I thought about this too, but as with so many python things, you should be able to set anything to whatever. But I see your point. Setting serial_number to something  else does not really change anything on the camera. Feels wrong with so many @properties, feels like I am writing C++ or something.

File: zivid/camera.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (trym-b) [reply zivid/camera.py]: fixed

**Pull request merged**